### PR TITLE
Add NUnit news for 2021-12

### PR DESCRIPTION
**Pull request by jnm2**
@ChrisMaddock, @CharliePoole, @OsirisTerje, @mikkelbu, @SeanKilleen, @rprouse

### CODE REVIEW

**Review by mikkelbu — APPROVED** [review]
LGTM

File: input/blog/posts/_current-newsletter-draft.md [hunk 1]
```diff
@@ -31,7 +31,14 @@ The .NET Foundation would like to welcome the following new projects!
 
 ### New Releases
 
+#### [NUnit](https://nunit.org)
 
+[NUnit Console & Engine 3.13](https://docs.nunit.org/articles/nunit/release-notes/console-and-engine.html#nunit-console--engine-313---november-30-2021) is the first release that allows running .NET Core/5+ tests, either separately or in combination with .NET Framework tests. A second major feature in this release is engine support for preemptive cancellation when the normal approach of requesting the test run to self-terminate doesn't work.
+
+[NUnit Adapter 4.1.0](https://docs.nunit.org/articles/vs-test-adapter/AdapterV4-Release-Notes#nunit3-test-adapter-for-visual-studio---version-410---nov-8-2021) and [4.2.0](https://docs.nunit.org/articles/vs-test-adapter/AdapterV4-Release-Notes#nunit3-test-adapter-for-visual-studio---version-420---dec-11-2021) fix exceptions during discovery and other bugs, such as explicit runs when using a negative category filter and AddTestAttachment not working in VS2022.
+
+[NUnit Analyzers 3.2 and 2.2](https://github.com/nunit/nunit.analyzers/releases/tag/3.2.0) contains an improvement to the suppression of
```
Comment (SeanKilleen) [inline input/blog/posts/_current-newsletter-draft.md:40]:
```suggestion
[NUnit Analyzers 3.2 and 2.2](https://github.com/nunit/nunit.analyzers/releases/tag/3.2.0) contain an improvement to the suppression of
```

**Review by SeanKilleen — APPROVED** [review]
LGTM. 👍 One minor nitpick. You also may want to consider using backticks around method names, error messages, etc. but I think it's fine either way.

Thanks for putting these together @jnm2!

**Review by Arhell — APPROVED** [review]

**Comment by jnm2** [comment]
@SeanKilleen I was on the fence about whether the visual noise would draw too much attention from the surrounding text, but I couldn't make up my mind and "doing less" broke the tie. You don't think so then?

**Comment by SeanKilleen** [comment]
@jnm2 as long as it's a conscious choice it seems fine to me! 👍

**Pull request merged**